chain-impl-mockchain/src/testing/builders/tx_builder.rs

```diff
@@ -128,8 +128,8 @@ impl TestTxBuilder {
         )];
         let tx_builder = TxBuilder::new()
             .set_payload(&NoExtra)
-            .set_expiry_date(validity.unwrap_or(BlockDate::first()))
-            .set_ios(&inputs, &destination);
+            .set_expiry_date(validity.unwrap_or_else(BlockDate::first))
```

Comment:
Not that it's related to this PR, but I found interesting that the default option is only valid if you are in fact building block 0